logical/plugin/system.go

```diff
@@ -0,0 +1,245 @@
+package plugin
+
+import (
+	"net/rpc"
+	"time"
+
+	"fmt"
+
+	plugin "github.com/hashicorp/go-plugin"
+	"github.com/hashicorp/vault/helper/consts"
+	"github.com/hashicorp/vault/helper/pluginutil"
+	"github.com/hashicorp/vault/helper/wrapping"
+	"github.com/hashicorp/vault/logical"
+)
+
+type SystemViewClient struct {
+	client *rpc.Client
+}
+
+func (s SystemViewClient) DefaultLeaseTTL() time.Duration {
```

Comment:
@briankassouf Should these methods be pointer receivers? I went with non-pointers because that's what `dynamicSystemView` and `StaticSystemView` implemented the `SystemView` interface.